docs/tutorials/cwp_calib_tutorial.rst

```diff
@@ -0,0 +1,185 @@
+===========================================================
+Borehole Calibration with Selective Simulation Cancellation
+===========================================================
+
+Introduction - Calibration with libEnsemble and CWP
+---------------------------------------------------
+
+This tutorial demonstrates libEnsemble's capability to selectively cancel pending
+simulations based on instructions from the *Persistent CWP* calibration
+generator function. This capability is critical for this calibration use-case since
+it isn't useful for the generator to receive extraneous evaluations
+from resources that may be more effectively applied towards critical evaluations.
+
+[JLN: BETTER JUSTIFICATION GOES HERE?]
+
+For a somewhat different approach than libEnsemble's :doc:`other tutorials<tutorials>`,
+we'll emphasize the settings, functions, and data fields within the calling script, CWP
+:ref:`persistent generator<persistent-gens>`, Manager, and :ref:`sim_f<api_sim_f>`
+that make this capability possible, rather than outlining a step-by-step process
+for writing this exact use-case.
+
+.. note::
+    The generator function featured in this tutorial can be found in ``gen_funcs/persistent_cwp_calib.py``. This version uses simplified standalone routines in place of the in-development CWP library.
+
```

Comment:
Wondering if maybe some psuedo-code of gen would help???? See what you think @jlnav @mosesyhc 
Something like

* Calculate test values and first batch
* While no STOP signal received:
  * Unpack evaluated points into 2D theta x point structures
  * If new model condition:
    * Construct new model
  * else
    * wait to receive more points
  * if some condition:
    * Generate new points from model
    * If threshold error is reached
      * Exit loop - done
  * If issued points are obviated
    * Cancel points

Could even show data layout, not sure if needed?

H:
 theta1, x1
 theta1, x2
 theta1, x3
 ...
 
->

fevals
 Row(theta 1):  x1, x2, x3, x4, x5
 Row(theta 2):  x1, x2, x3, x4, x5
 Row(theta 3):  x1, x2, x3, x4, x5
 ...

failures, and data_status same layout.